libraries/AP_InertialSensor/AP_InertialSensor_RST.cpp

```diff
@@ -0,0 +1,436 @@
+#define GYRO_WHO_AM_I			0x0F
+#define GYRO_WHO_I_AM			0xD3
+
+#define GYRO_CTRL_REG1			0x20
+/* keep lowpass low to avoid noise issues */
+#define RATE_100HZ_LP_25HZ		((0<<7) | (0<<6) | (0<<5) | (1<<4))
+#define RATE_200HZ_LP_25HZ		((0<<7) | (1<<6) | (0<<5) | (1<<4))
+#define RATE_200HZ_LP_50HZ		((0<<7) | (1<<6) | (1<<5) | (0<<4))
+#define RATE_200HZ_LP_70HZ		((0<<7) | (1<<6) | (1<<5) | (1<<4))
+#define RATE_400HZ_LP_20HZ		((1<<7) | (0<<6) | (1<<5) | (0<<4))
+#define RATE_400HZ_LP_25HZ		((1<<7) | (0<<6) | (0<<5) | (1<<4))
+#define RATE_400HZ_LP_50HZ		((1<<7) | (0<<6) | (1<<5) | (0<<4))
+#define RATE_400HZ_LP_100HZ		((1<<7) | (0<<6) | (1<<5) | (1<<4))
+#define RATE_800HZ_LP_30HZ		((1<<7) | (1<<6) | (0<<5) | (0<<4))
+#define RATE_800HZ_LP_35HZ		((1<<7) | (1<<6) | (0<<5) | (1<<4))
+#define RATE_800HZ_LP_50HZ		((1<<7) | (1<<6) | (1<<5) | (0<<4))
+#define RATE_800HZ_LP_100HZ		((1<<7) | (1<<6) | (1<<5) | (1<<4))
+
+#define GYRO_CTRL_REG2			0x21
+#define GYRO_CTRL_REG3			0x22
+#define GYRO_CTRL_REG4			0x23
+#define RANGE_250DPS            (0<<4)
+#define RANGE_500DPS            (1<<4)
+#define RANGE_2000DPS           (3<<4)
+
+#define GYRO_CTRL_REG5			0x24
+#define GYRO_REFERENCE			0x25
+#define GYRO_OUT_TEMP			0x26
+#define GYRO_STATUS_REG			0x27
+#define GYRO_OUT_X_L			0x28
+#define GYRO_OUT_X_H			0x29
+#define GYRO_OUT_Y_L			0x2A
+#define GYRO_OUT_Y_H			0x2B
+#define GYRO_OUT_Z_L			0x2C
+#define GYRO_OUT_Z_H			0x2D
+#define GYRO_FIFO_CTRL_REG		0x2E
+#define GYRO_FIFO_SRC_REG		0x2F
+#define GYRO_INT1_CFG			0x30
+#define GYRO_INT1_SRC			0x31
+#define GYRO_INT1_TSH_XH		0x32
+#define GYRO_INT1_TSH_XL		0x33
+#define GYRO_INT1_TSH_YH		0x34
+#define GYRO_INT1_TSH_YL		0x35
+#define GYRO_INT1_TSH_ZH		0x36
+#define GYRO_INT1_TSH_ZL		0x37
+#define GYRO_INT1_DURATION		0x38
+#define GYRO_LOW_ODR			0x39
+
+
+/* Internal configuration values */
+#define GYRO_REG1_POWER_NORMAL			(1<<3)
+#define GYRO_REG1_Z_ENABLE				(1<<2)
+#define GYRO_REG1_Y_ENABLE				(1<<1)
+#define GYRO_REG1_X_ENABLE				(1<<0)
+
+#define GYRO_REG4_BLE                   (1<<6)
+
+#define GYRO_REG5_FIFO_ENABLE			(1<<6)
+#define GYRO_REG5_REBOOT_MEMORY			(1<<7)
+
+#define GYRO_STATUS_ZYXOR				(1<<7)
+#define GYRO_STATUS_ZOR                 (1<<6)
+#define GYRO_STATUS_YOR                 (1<<5)
+#define GYRO_STATUS_XOR                 (1<<4)
+#define GYRO_STATUS_ZYXDA				(1<<3)
+#define GYRO_STATUS_ZDA                 (1<<2)
+#define GYRO_STATUS_YDA                 (1<<1)
+#define GYRO_STATUS_XDA                 (1<<0)
+
+#define GYRO_FIFO_CTRL_BYPASS_MODE			    (0<<5)
+#define GYRO_FIFO_CTRL_FIFO_MODE			    (1<<5)
+#define GYRO_FIFO_CTRL_STREAM_MODE			    (1<<6)
+#define GYRO_FIFO_CTRL_STREAM_TO_FIFO_MODE	    (3<<5)
+#define GYRO_FIFO_CTRL_BYPASS_TO_STREAM_MODE    (1<<7)
+
+#define GYRO_DEFAULT_RATE               800 //data output frequency
+#define GYRO_DEFAULT_RANGE_DPS          2000
+#define GYRO_DEFAULT_FILTER_FREQ		35
+#define GYRO_TEMP_OFFSET_CELSIUS		40
```

Comment:
Please don't use tabs, only spaces. As you can see your macro values alignment looks strange, seems like there is no consistency.

By the way, when a macro is sub-related to a macro above it we usually indent the entire line instead of the value, like:

```
#define ACCEL_STATUS_ZYXOR  (1<<7)
   #define ACCEL_STATUS_ZOR  (1<<6)
   #define ACCEL_STATUS_YOR  (1<<5)
   #define ACCEL_STATUS_XOR  (1<<4)
```